src/elements/content-sidebar/activity-feed/app-activity/AppActivity.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../../common/variables';
+
+.be {
+    .bcs-app-activity-container {
+        position: relative;
+        z-index: 1;
+
+        &:hover,
+        &.bcs-is-focused {
+            .bcs-inline-delete-container {
+                opacity: 1;
+            }
+        }
+
+        .bcs-app-activity {
+            display: flex;
+            margin-top: 23px;
+            opacity: .9999;
+            width: 100%;
+
+            &.bcs-is-pending {
+                opacity: .4;
+            }
+
+            .bcs-app-activity-icon {
+                flex-shrink: 0;
+                height: 32px;
+                margin-right: 10px;
+                width: 32px;
+            }
+
+            .bcs-app-activity-content {
+                color: $fours;
+                flex: 1;
```

Comment:
Shorthand flex values can cause issues in IE and Safari. Is this `1 1 auto`, `1 0 auto`, `1 1 100%`, etc.?